pkg/service/manipulator.go

```diff
@@ -95,9 +101,9 @@ func (m *manipulator) Process(spec ProcessSpec) ([]byte, error) {
 		data = m.processor.Flip(data, params[flip])
 		trackDuration(flipDurationKey, t, spec)
 	}
-	if CleanAngle(params[rotate]) > 0 {
+	if CleanFloat(params[rotate], 360) > 0 {
```

Comment:
can be improved so that we don't need to call CleanFloat twice:
if angle := CleanFloat(params[blur], 360); angle > 0